hail/www/index.md

```diff
@@ -1,63 +1,82 @@
 # Hail
 
-Hail is an open-source, scalable framework for exploring and analyzing genomic data. 
+[Hail](https://hail.is) is an open-source, general-purpose, Python-based data analysis tool with additional data types and methods for working with genomic data.
 
-For genomics applications, Hail can, for example:
+Hail is built to scale and has first-class support for multi-dimensional structured data, like the genomic data in a genome-wide association study (GWAS).
 
- - flexibly [import and export](https://hail.is/docs/0.2/methods/impex.html) to a variety of data and annotation formats, including [VCF](https://samtools.github.io/hts-specs/VCFv4.2.pdf), [BGEN](http://www.well.ox.ac.uk/~gav/bgen_format/bgen_format_v1.2.html) and [PLINK](https://www.cog-genomics.org/plink2/formats)
- - generate variant annotations like call rate, Hardy-Weinberg equilibrium p-value, and population-specific allele count; and import annotations in parallel through [annotation datasets](https://hail.is/docs/stable/datasets.html), [VEP](https://useast.ensembl.org/info/docs/tools/vep/index.html), and [Nirvana](https://github.com/Illumina/Nirvana/wiki)
- - compute sample annotations like mean depth, imputed sex, and TiTv ratio
- - compute new annotations from existing ones as well as genotypes, and use these to filter samples, variants, and genotypes
- - find Mendelian violations in trios, prune variants in linkage disequilibrium, analyze genetic similarity between samples, and compute sample scores and variant loadings using PCA
- - perform variant, gene-burden and eQTL association analyses using linear, logistic, Poisson, and linear mixed regression, and estimate heritability
- - lots more! Check out some of the new features in [Hail 0.2](http://discuss.hail.is/t/announcing-hail-0-2/702/1).
+Hail is exposed as a Python library, using primitives for distributed queries and linear algebra implemented in Scala, [Spark](https://spark.apache.org/docs/latest/index.html), and increasingly C++.
 
-Hail is a **[Python](https://www.python.org/)**  library with a scalable backend built on top of **[Apache Spark](https://spark.apache.org/docs/latest/index.html)** to efficiently analyze gigabyte-scale data on a laptop or terabyte-scale data on a cluster. 
+See the [documentation](https://hail.is/docs/0.2/) for more info on using Hail.
 
-### Getting Started
+### Contribute
 
-To get started using Hail:
+If you'd like to discuss or contribute to the development of methods or infrastructure, please:
 
-- install Hail 0.2 using the instructions in [Installation](https://hail.is/docs/0.2/getting_started.html)
```

Comment:
I think we should keep some of these bullets after "See the documentation" to help users understand what's there and navigate to the right place. (When we split the readme from the index, we also split bullets for users and contributors there)

Reply:
Con: they must click the documentation link
Pro: the links don't break

I kind of prefer not calling out these doc sections here and on the docs landing page. Thoughts?

Reply:
the documentation landing page is shit. we need to link somewhere other than that, or fix it

Reply:
I agree with Tim. In the meantime, Dan, you should at least bullet what's not in the docs: the user chat and user forum